frontend/src/components/IssueModeSelector/IssueModeSelector.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Text } from '@highlight-run/ui/components'
+import clsx from 'clsx'
+
+import styles from './IssueModeSelector.module.css'
+
+export const TagSwitch = <T extends string | number>({
+	onSelect,
+	label,
+	selected,
+}: {
+	onSelect: (p: T) => void
+	label: any
+	key: any
+	selected: boolean
+}) => {
+	const modeClass = selected ? styles.activeSwitch : styles.inactiveSwitch
+
+	return (
+		<div
+			className={clsx(styles.tagSwitch, modeClass)}
```

Comment:
Nit, but we try to use the object syntax when conditionally applying classes like this. A little more verbose, but maybe a little more readable.

```tsx
clsx(styles.tagSwitch, {
  [styles.activeSwitch]: selected,
  [styles.inactiveSwitch]: !selected,
})}
```

Reply:
Also, wondering if we could use `Box` here and apply many of these styles via props rather than using a separate stylesheet. Not a big deal, just a thought for future contributions 😄